app/controllers/admin/community_memberships_controller.rb

```diff
@@ -91,6 +93,8 @@ def generate_csv_for(memberships)
             user.given_name,
```

Comment:
Since find_each sorts by id ASC, the end result is the same as with sorting by creation date. Went ahead with find_each.